typescript/src/entities/variation_fields_option.test.ts

```diff
@@ -5,3 +5,13 @@ test('can make VariationFieldsOption', () => {
   const variationFieldsOption = new merchi.VariationFieldsOption();
   expect(variationFieldsOption).toBeTruthy();
 });
+
+test('totalCost', () => {
+  const merchi = new Merchi();
+  const vfo = new merchi.VariationFieldsOption();
+  expect(vfo.totalCost).toThrow();
```

Comment:
just looks like a little bit magic to me

why you do not need call it vfo.totalCost

Also suggest to check the error throwed is what we expected to throw.

Reply:
ok I see the it is actually the same to vfo.totalCost(undefined), I think in real world scenario this actually should cause a type error rather then the error we expected.

for is unknown test it as expect (() => {vfo.totalCost(33)}).